frontend/src/app/AppLauncher.tsx

```diff
@@ -127,42 +119,53 @@ const AppLauncher: React.FC = () => {
 
   const renderApplicationLauncherGroup = (section: Section, sectionIndex: number) => {
     const appItems = section.actions.map((action) => (
-      <ApplicationLauncherItem
+      <DropdownItem
```

Comment:
The PF example nests items within `<DropdownList>` but that's not done here. Any specific reason?